tests/cypress/e2e/api-spec.cy.js

```diff
@@ -0,0 +1,62 @@
+describe('DiAGRAM API spec', () => {
+    it('should get the model score when creating a new model', () => {
+        cy.request({
+            method: 'POST',
+            url: '/api/model/score',
+            body: {"model_name":"fdg","scenario":"Base Model","notes":"","is_advanced":false,"intellectual_control":null,"renderability":null,"response":{"Digital_Object":[0,0,100],"Storage_Medium":[0,0,100],"Rep_and_Refresh":{"1":0,"2":0},"Op_Environment":{"1":0,"2":"Not Applicable - we have copies offsite"},"Physical_Disaster":"Very Low","Checksum":[0,0,100],"System_Security":{"1":"ISO 27001","2":"None, or only minor issues outstanding","3":"Level 4","4":"Yes"},"Info_Management":{"1":"Not achieved","2":"Not achieved","3":["Minimal awareness","Minimal awareness"]},"Technical_Skills":["None","None","None","None","None","None","None","None","None","None"]},"advanced":null},
+            headers: {
+                'Content-type': 'application/json; charset=UTF-8',
+            },
+        }).then((response) => {
+            expect(response.status).to.eq(200);
+            expect(response.body).to.include({"renderability": 44.679});
+            expect(response.body).to.have.property('nodes');
+            expect(response.body.nodes).to.have.property('Op_Environment');
+            expect(response.body.nodes.Op_Environment).to.include({Yes: 0, No: 1});
+        });
+    });
+
+    it('uploading an invalid model should succeeed and report the model to be invalid', () => {
```

Comment:
1. Since the test starts with "it", should this test be "should succeed in uploading an invalid model and report the model to be invalid" or "should succeed in uploading an invalid model and report it as invalid"?
2. You have 3 "e"s in "succeed"